src/arrayMethodLastIndexOf.js

```diff
@@ -4,8 +4,33 @@
  * Implement method lastIndexOf
  */
 function applyCustomLastIndexOf() {
-  [].__proto__.lastIndexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  [].__proto__.lastIndexOf2 = function(searchElement,
+    fromIndex = this.length - 1) {
```

Comment:
так не переносять код
```suggestion
  [].__proto__.lastIndexOf2 = function(
    searchElement,
    fromIndex = this.length - 1,
  ) {
```

Reply:
Виправив. Дякую.